common/common.go

```diff
@@ -102,6 +102,11 @@ func PodManifestPath(root string) string {
 	return filepath.Join(root, "pod")
 }
 
+// PodManifestLockPath returns the path in root to the Pod Manifest lock file
+func PodManifestLockPath(root string) string {
```

Comment:
We are `os.Rename`ing the original pod manifest file to make the change atomic, hence we need a different lock file (or directory). Wondering were to document this fact though ... I'll do it here, and in the lifecycle document.